lib/sms/sms_at.h

```diff
@@ -7,6 +7,9 @@
 #ifndef _SMS_AT_INCLUDE_H_
 #define _SMS_AT_INCLUDE_H_
 
+/** @brief Error code for AT command that is not SMS related. */
+#define ENOTSMSAT 0xFFFFFF00
```

Comment:
This value should be smaller because the returned value is a signed integer.

Reply:
Changed this to 0x0FFFFFFF